rhobs/alerting/data_plane/prometheus.release_service_alerts.yaml

```diff
@@ -59,3 +59,21 @@ spec:
         alert_team_handle: <!subteam^S03SVBS426R>
         runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/release-service/release-processing-latency.md?ref_type=heads
         team: release
+
+    - alert: ReleaseServiceControllerManagerAvailability
+      expr: |
+        rate(
+            sum( (sum(kube_pod_container_status_ready{namespace="release-service", container="manager"}) by(job) OR on() vector(0))) [10m:]
```

Comment:
the comments a related. This metric give 1 or absent in case of pod not ready. To calculate the rate, I need it to be when absent, so the `vector` needs to be added. But this creates an extra item without labels, for instance:

when the data exists (or with 1)
```
kube_pod_container_status_ready{namespace="release-service"}  1
{}          0
```

About the corner case, that's the one explained  - if no data (which is the case when the pod is down) it gives 0.

I will look up how to emulate this in the test.

Reply:
I am trying to simplify the query using other metrics now that they are available.

Reply:
I fully changed the query to be more intuitive.